reference/dtr/2.7/api/index.html

```diff
@@ -1,16 +1,18 @@
+---
+layout: null
```

Comment:
Not sure if this is still needed, I think this was to prevent Jekyll producing errors when trying to convert html files (it would otherwise attempt to apply the default layout)